packages/worker/src/workflow/vm.ts

```diff
@@ -256,7 +255,27 @@ export class VMWorkflowCreator implements WorkflowCreator {
       });
     }
 
-    return new this(script, sourceMap, isolateExecutionTimeoutMs) as InstanceType<T>;
+    return new this(script, isolateExecutionTimeoutMs) as InstanceType<T>;
+  }
+
+  private static async makeScriptWithSourceMap(code: string) {
+    const sourceMappingUrlDataRegex = /\s*\n[/][/][#]\s+sourceMappingURL=data:(?:[^,]*),([^\s]+)\s*$/m;
+    const sourceMapMatcher = code.match(sourceMappingUrlDataRegex);
+
+    if (sourceMapMatcher) {
+      const sourceMap = Buffer.from(sourceMapMatcher[1], 'base64').toString();
+      const sourceMapParsed: RawSourceMap = JSON.parse(sourceMap);
+
+      const filename = `/temporal/${sourceMapParsed.file}`;
```

Comment:
Why `/temporal`?

Reply:
I think this will be a problem on windows.

Reply:
After more testing, it is much more reliable if I use `cwd()`. That also resolve the Windows concern.